src_cpp/opennnBridge/tmp.cpp

```diff
@@ -0,0 +1,18 @@
+
+
+#include "niftest.h" 
+#include "nifppEigenExtensions.h"
+/*#include <vector>
+#include "opennn.h"
+
+using namespace OpenNN;*/
+
+// I am another branch
+
+int main(void)
```

Comment:
this file can be switched to test, we can also add it as a project in our cmake and generate execute for it if you want 
@evgenyan95